### PR TITLE
*: refine docs about binlog GTID

### PR DESCRIPTION
**Pull request by csuzhangxc**
### What is changed, added or deleted? (Required)

- refine comments for `enable-gtid: false`
- add `binlog-gtid` for task config

### Which DM version(s) do your changes apply to? (Required)

- [x] master (the latest development version, including v3.0 changes)
- [x] v2.0 (TiDB DM 2.0 versions)
- [ ] v1.0 (TiDB DM 1.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Have version specific changes 
- [ ] Might cause conflicts after cherry-picked

### CODE REVIEW

**Comment by csuzhangxc** [comment]
@lance6716 @ireneontheway PTAL

**Review by lance6716 — COMMENTED** [review]
LGTM

File: en/task-configuration-file-full.md [hunk 1]
```diff
@@ -121,6 +121,7 @@ mysql-instances:
 
       binlog-name: binlog.000001
       binlog-pos: 4
+      binlog-gtid: "03fc0263-28c7-11e7-a653-6c0b84d59f30:1-7041423,05474d3c-28c7-11e7-8352-203db246dd3d:1-170"  # Set this argument if specified `enable-gtid: true` for the source of the incremental task.
```
Comment (ireneontheway) [inline en/task-configuration-file-full.md:124]:
```suggestion
      binlog-gtid: "03fc0263-28c7-11e7-a653-6c0b84d59f30:1-7041423,05474d3c-28c7-11e7-8352-203db246dd3d:1-170"  # For incremental tasks with `enable-gtid: true` specified in the source, set this argument.
```

**Review by yikeke — APPROVED** [review]
Rest LGTM

**Comment by csuzhangxc** [comment]
@yikeke PTAL again

**Review by yikeke — APPROVED** [review]
LGTM

**Pull request merged**